packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -35,15 +37,16 @@ class InspectorDetailsTabController extends StatelessWidget {
   Widget build(BuildContext context) {
     final tabs = <Tab>[
       _buildTab('Details Tree'),
-      _buildTab('Layout Details'),
+      if (layoutDetailsSupported) _buildTab('Layout Details'),
     ];
     final tabViews = <Widget>[
       detailsTree,
-      Banner(
-        message: 'PROTOTYPE',
-        location: BannerLocation.topStart,
-        child: LayoutDetailsTab(controller: controller),
-      ),
+      if (layoutDetailsSupported)
+        Banner(
```

Comment:
can we remove the prototype banner now?

Reply:
do we want to keep it in the demo? @DaveShuckerow